package.json

```diff
@@ -22,10 +25,5 @@
     "eslint-plugin-react-hooks": "^4.2.0",
     "npm-run-all": "^4.1.5"
   },
-  "dependencies": {
```

Comment:
Backend dependencies are already in the server/package.json, so removed them from here.